src/Oci8/Oci8Connection.php

```diff
@@ -216,9 +216,10 @@ protected function getDoctrineDriver()
      * @param string $sql (mixed)
      * @param array $bindings (kvp array)
      * @param int $returnType (PDO::PARAM_*)
+     * @param int $length
      * @return mixed $returnType
      */
-    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR)
+    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR, $length = null)
```

Comment:
i think null is ok 
reference [PDO::bindParam](https://github.com/JetBrains/phpstorm-stubs/blob/master/PDO/PDO.php#L1061)